src/solvers/cma_es.jl

```diff
@@ -0,0 +1,710 @@
+#
+# State
+#
+@doc raw"""
+    CMAESState{P,T} <: AbstractManoptSolverState
+
+State of covariance matrix adaptation evolution strategy.
+"""
```

Comment:
It doesn't really have any special constructor right now, all construction is done directly in `cma_es`, though maybe that's not ideal. Regarding documentation of fields, do you want me then to remove the docs in comments? It was easier to deal with when writing the code because I had only one list of fields instead of two that I had to keep consistent (one with docs, one in struct definition).

Reply:
I commented that at that point as well. A specific constructor would be nice I think.

For the docs – yes please more the comments to a docstring, it makes the documentation more accessible (for example with the live docs in Pluto) – and one has not to open the source code to read it. Sure during design it might be easier to sketch that in comments after the fields, but for the final version, doc strings would be nicer.